external/anomaly/tests/test_ote_training.py

```diff
@@ -0,0 +1,339 @@
+            f'ERROR: dataset definitions for name={dataset_name} contains empty values for some required fields'
+
+    params = DatasetParameters(**training_parameters_fields)
+    return params
+
+
+def _create_anomaly_classification_dataset_and_labels_schema(dataset_params):
+    logger.debug(f'Path to dataset: {dataset_params.dataset_path}')
+    category_list = [f.path for f in os.scandir(dataset_params.dataset_path) if f.is_dir()]
+    items = []
+    for category in category_list:
+        if "vivo" not in category:
+            logger.debug(f'Creating dataset for {category}')
+            items.extend(OteMvtecDataset(path=category, seed=0).generate())
+    dataset = DatasetEntity(items=items)
+    labels = dataset.get_labels()
+    labels_schema = LabelSchemaEntity.from_labels(labels)
+    return dataset, labels_schema
+
+
+def get_anomaly_test_action_classes() -> List[Type[BaseOTETestAction]]:
+    return [
+        AnomalyDetectionTestTrainingAction,
+        OTETestTrainingEvaluationAction,
+        OTETestExportAction,
+        OTETestExportEvaluationAction,
+        OTETestPotAction,
+        OTETestPotEvaluationAction,
+        OTETestNNCFAction,
+        OTETestNNCFEvaluationAction,
+        OTETestNNCFExportAction,
+        OTETestNNCFExportEvaluationAction,
+    ]
+
+
+class AnomalyDetectionTrainingTestParameters(DefaultOTETestCreationParametersInterface):
+    def test_case_class(self) -> Type[OTETestCaseInterface]:
+        return generate_ote_integration_test_case_class(
+            get_anomaly_test_action_classes()
+        )
+
+    def test_bunches(self) -> List[Dict[str, Any]]:
+        test_bunches = [
+                dict(
+                    model_name=[
+                       'ote_anomaly_classification_padim',
+                       'ote_anomaly_classification_stfpm',
+                    ],
+                    dataset_name='mvtec',
+                    usecase='precommit',
+                ),
+                dict(
+                    model_name=[
+                       'ote_anomaly_classification_padim',
+                       'ote_anomaly_classification_stfpm',
+                    ],
+                    dataset_name='mvtec',
+                    batch_size=KEEP_CONFIG_FIELD_VALUE,
+                    usecase=REALLIFE_USECASE_CONSTANT,
+                ),
+
+        ]
+        return deepcopy(test_bunches)
+    
+    def short_test_parameters_names_for_generating_id(self) -> OrderedDict:
+        DEFAULT_SHORT_TEST_PARAMETERS_NAMES_FOR_GENERATING_ID = OrderedDict(
+            [
+                ("test_stage", "ACTION"),
+                ("model_name", "model"),
+                ("dataset_name", "dataset"),
+                ("batch_size", "batch"),
+                ("usecase", "usecase"),
+            ]
+        )
+        return deepcopy(DEFAULT_SHORT_TEST_PARAMETERS_NAMES_FOR_GENERATING_ID)
+
+    def test_parameters_defining_test_case_behavior(self) -> List[str]:
```

Comment:
Do you really need to override this method too?
Is it different w.r.t. `DefaultOTETestCreationParametersInterface`?

Reply:
P.S. Please, check it for other methods of the class.